include/zephyr/drivers/tee.h

```diff
@@ -349,7 +349,7 @@ typedef int (*tee_shm_unregister_t)(const struct device *dev, struct tee_shm *sh
  *
  * See @a tee_suppl_recv() for argument definitions.
  */
-typedef int (*tee_suppl_recv_t)(const struct device *dev, uint32_t func, unsigned int num_params,
+typedef int (*tee_suppl_recv_t)(const struct device *dev, uint32_t *func, unsigned int *num_params,
```

Comment:
Changes in this file should go into a separate commit with a note that they should be squashed into original commit during upstreaming process